views/index.html

```diff
@@ -2,8 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Lecture Viewer</title>
-    <link rel="stylesheet" href="style.css">
+    <title>Lecture Viewer do Kapiroto</title>
+    <link type="text/css" rel="stylesheet" href="style/style.css">
+    <link type="text/css" rel="stylesheet" href="style/bootstrap.css">
+    <script src="/style/bootstrap.js"></script>
```

Comment:
I want to move away from bootstrap once we have an initial prototype, it's an awful framework. But this is something I need to discuss with mary.